daemon/health.go

```diff
@@ -136,9 +136,16 @@ func (p *cmdProbe) run(ctx context.Context, d *Daemon, cntr *container.Container
 		// Wait for probe to exit (it might take a while to respond to the TERM
 		// signal and we don't want dying probes to pile up).
 		<-execErr
+		out := output.String()
+		var msg string
+		if len(out) > 0 {
+			msg = fmt.Sprintf("Health check exceeded timeout (%v):\n%s", probeTimeout, out)
```

Comment:
Before I forget to ask; I see this adds a newline before the output; was there a specific reason for that? Or was it more readable if the output contained multiple lines (e.g.)?

Was wondering if it'd be more readable (assuming single line output) to have it on the same line (`..exceeded timeout(): <output>`) to look more like a regular error message, but perhaps that didn't work well, so thought I'd ask

Reply:
no specific reason, feels more natural for me to add timeout message as a separate line, then command output